main.go

```diff
@@ -77,6 +80,390 @@ func main() {
 					return nil
 				},
 			},
+			{
+				Name:  "create-customer",
+				Usage: "Creates a new customer to go in the customers database, must specify email and state(2 letter code)",
+				Action: func(cCtx *cli.Context) error {
+					if len(os.Args) != 4 {
+						return errors.New("Must specify email and state")
+					}
+					email := os.Args[2]
+					state := os.Args[3]
+					insertStatement := "INSERT INTO Customers (email, state) VALUES (?, ?)"
+					_, err := db.Exec(insertStatement, email, state)
```

Comment:
Note that the result return type here (https://pkg.go.dev/database/sql#Result) actually holds the last insert id:
```suggestion
					res, err := db.Exec(insertStatement, email, state)
					...
					id, err := res.LastInsertId()
```